app/authentication/auth.py

```diff
@@ -18,7 +18,10 @@
 
 from app.serialised_models import SerialisedService
 
-GENERAL_TOKEN_ERROR_MESSAGE = 'Invalid token: make sure your API token matches the example at https://docs.notifications.service.gov.uk/rest-api.html#authorisation-header'  # nosec B105
+GENERAL_TOKEN_ERROR_MESSAGE = '''
+        Invalid token: make sure your API token matches the example
+        at https://docs.notifications.service.gov.uk/rest-api.html#authorisation-header
+    '''  # nosec B105
```

Comment:
without checking the failure, I'd bet that this multi-line string is the cause of the static-scan failure.

Reply:
Haha, yep, that's exactly it

Reply:
Okay, rebased to pull in the `pip-audit` fix and uncommented the github action, if you'd like to take another look